docker-compose.yml

```diff
@@ -55,9 +57,11 @@ services:
     container_name: rems_prod_rems
     ports: # Port binding to host from docker container
       - "8090:8090"
-    environment:
-      VSAC_API_KEY: ${VSAC_API_KEY}
-      MONGO_URL: mongodb://rems-user:pass@pims_remsadmin_mongo:27017
```

Comment:
these should be reverted back to environment instead of .env.json as MONGO_URL is an override to connect to mongo from the container itself, which is different then if you ran locally without docker (which is the default value). Same goes for the VSAC_API_KEY which needs to be specified by the user in their environment but can also be done so in many ways. Similar reason to using env_file in rems-smart-on-fhir (see comments in the other REMS PR)